test/parallel/test-buffer.js

```diff
@@ -1476,3 +1476,26 @@ assert.equal(SlowBuffer.prototype.offset, undefined);
   // Check pool offset after that by trying to write string into the pool.
   assert.doesNotThrow(() => Buffer.from('abc'));
 }
+
+
+// Test failed or zero-sized Buffer allocations not affecting typed arrays
+{
+  const zeroArray = new Uint32Array(10).fill(0);
```

Comment:
This testcase ensures that typed arrays are zero filled, by comparing them to a typed array that is surely zero-filled.

If something breaks and typed arrays become non zero filled, then this testcase should fail. Without `.fill(0)` a change where this and following typed arrays become filled with equivalent garbage (e.g. with a constant number due to some of the previous tests) will slip through.